intro-HPC/ch_account.tex

```diff
@@ -351,11 +351,13 @@ \subsection{Users of the \association}
 \fi % END IF ANTWERPEN
 
 \ifgent
-  You will now have to log in with CAS using your UGent account, and then be requested
-  to share your Mail and Affiliation with KU~Leuven (which actually means the VSC accountpage).
-  Select the ``Remember'' check-box and click ``Yes, continue''.
+  You will now have to log in with CAS using your UGent account. You either have a
+  login name of maximum 8 characters, or a (non-UGent) email address if you are an external
+  user. In case of problems with your UGent password, please visit:
+  \url{https://password.ugent.be/}. After logging in, you will be requested
+  to share your information. Click ``Yes, continue''.
```

Comment:
Break this up in paragraph, people lost interest from the 2nd sentence onwards, so important stuff needs to be in the first sentence. ;)

Also, the last sentence is worded too strongly, change `will` to `may be` (you may have approved it already recently)